examples/with-edgedb/globals.ts

```diff
@@ -0,0 +1,3 @@
+export const ROOT_URL = process.env.VERCEL_URL
```

Comment:
NEXT_PUBLIC_VERCEL_URL

Reply:
Although with other changes to `fetch()` calls we probably should just drop this file.